addon/components/search-facet-worktype-button/template.hbs

```diff
@@ -0,0 +1,16 @@
+{{!Adapted from Ember-SHARE}}
+<div class='type-filter-option {{if selected 'active'}}'>
+    {{#if collapsible}}
+        <button class="btn-link" aria-label={{t 'eosf.components.searchFacetWorktypeButton.expandWorktype'}}>
+            <i class='fa expand-icon clickable {{if toggleState 'fa-plus' 'fa-minus'}}' {{action 'toggleBody'}}></i>
+        </button>
+    {{/if}}
+    <span class={{if selected 'bubble'}}>
+        <button class='clickable btn-link' {{action 'click'}}>{{label}}</button>
+        {{#if selected}}
+            <button class="btn-link" aria-label={{t 'eosf.components.searchFacetWorktypeButton.removeWorktypeFilter'}}>
+                <i class='fa fa-times close-icon clickable' {{action 'click'}}></i>
```

Comment:
Click action handler might go better on the `button` than the `i`con.

That said, thank you for setting me up for that HTML syntax pun.